.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,112 @@
+# Based on the analogous pypa/setuptools form.
+---
+name: 🐜 Bug report
+description: >-
+  Create a report to help us improve when
+  something is not working correctly.
+title: 'Bug: '
+labels:
+- bug
+
+body:
+- type: markdown
+  attributes:
+    value: >
+      **Thank you for reporting a bug in XLConnect!**
+      
+
+      ⚠
+      Verify first that your issue is not
+      [already reported on GitHub][issue search].
+
+      Also test if the latest CRAN release is affected.
+
+      If you are seeking community support, please consider
+      [asking a question on stackoverflow][stackoverflow].
+
+
+
+      Thank you for your collaboration!
+
+
+      [stackoverflow]: https://stackoverflow.com/questions/tagged/xlconnect
+
+      [issue search]: https://github.com/miraisolutions/xlconnect/search?q=is%3Aissue&type=issues
+
+- type: markdown
+  attributes:
+    value: >-
+      **Environment**
+- type: textarea
+  attributes:
+    label: sessionInfo() output
+    description: >-
+      Output of running sessionInfo in your r session with XLConnect loaded

```

```suggestion
      Output of running sessionInfo in your R session with XLConnect loaded.

```
I also added a final period `.`  for consistency